packages/core/src/storage/vectorStore/QdrantVectorStore.ts

```diff
@@ -289,7 +289,7 @@ export class QdrantVectorStore implements VectorStore {
    * @param query The VectorStoreQuery to be used
    */
   private async buildQueryFilter(query: VectorStoreQuery) {
-    if (!query.docIds && !query.queryStr) {
+    if (!query.docIds && !query.queryStr && !query.filters) {
```

Comment:
Did you test this without `filters` being specified?

Reply:
Yes, it does work. I can update the example to showcase that as well if needed

Reply:
If you have time, that would be great of course!

Reply:
Updated in the new commit